iosxr_iso2vbox.py

```diff
@@ -232,8 +226,8 @@ def verboseprint(*args):
 
     # Remove stale SSH entry
     verboseprint('Removing stale SSH entries')
-    run('ssh-keygen -R [127.0.0.1]:2222')
-    run('ssh-keygen -R [127.0.0.1]:2223')
+    run('ssh-keygen -R [localhost]:2222')
+    run('ssh-keygen -R [localhost]:2223')
```

Comment:
Just a thought, unrelated to this PR - if the user is using a multi-node Vagrantfile, they'll still have some stale entries for the other nodes, right? Is there a way we can hardcode the SSH keys on the XR side rather than generating them dynamically so that this is no longer an issue?